website/docs/VisualTesting.md

```diff
@@ -0,0 +1,281 @@
+                },
+                // ... more options
+            },
+        ],
+    ],
+    // ...
+};
+```
+
+More service options can be found [here](./service-options).
+
+### WebdriverIO MultiRemote
+
+We also support [MultiRemote](https://webdriver.io/docs/multiremote/). To make this work properly make sure that you add `wdio-ics:options` to your
+capabilities as you can see below. This will make sure that each screenshot will have its own unique name.
+
+[Writing your tests](./visual-testing/writing-tests) will not be any different in comparison to using the [TestRunner](https://webdriver.io/docs/testrunner)
+
+```js
+exports.config = {
+    capabilities: {
+        chromeBrowserOne: {
+            capabilities: {
+                browserName: "chrome",
+                "goog:chromeOptions": {
+                    args: ["disable-infobars"],
+                },
+                // THIS!!!
+                "wdio-ics:options": {
+                    logName: "chrome-latest-one",
+                },
+            },
+        },
+        chromeBrowserTwo: {
+            capabilities: {
+                browserName: "chrome",
+                "goog:chromeOptions": {
+                    args: ["disable-infobars"],
+                },
+                // THIS!!!
+                "wdio-ics:options": {
+                    logName: "chrome-latest-two",
+                },
+            },
+        },
+    },
+};
+```
+
+### Running Programmatically
+
+Here is a minimal example of how to use `wdio-image-comparison-service` via `remote` options
+
+```js
+import { remote } from "webdriverio";
+
+import WdioImageComparisonService from "wdio-image-comparison-service";
+
+let wdioImageComparisonService = new WdioImageComparisonService({});
+
+async function main() {
+    const browser = await remote({
+        logLevel: "silent",
+        capabilities: {
+            browserName: "chrome",
+        },
+    });
+
+    wdioImageComparisonService.defaultOptions.autoSaveBaseline = true;
+    browser.defaultOptions = wdioImageComparisonService.defaultOptions;
+    browser.folders = wdioImageComparisonService.folders;
+
+    wdioImageComparisonService.before(browser.capabilities);
+
+    await browser.url("https://webdriver.io/");
+
+    // or use this for ONLY saving a screenshot
+    await browser.saveFullPageScreen("examplePaged", {});
+
+    // or use this for validating. Both methods don't need to be combined, see the FAQ
+    await browser.checkFullPageScreen("examplePaged", {});
+
+    await browser.deleteSession();
+}
+
+main().catch(async (e) => {
+    console.error(e);
+});
+```
+
+### DEV-TOOLS support
+
+You can also use the Chrome DevTools as an automation protocol in combination with this module. You don't need to do anything,
+just change `automationProtocol: 'devtools'` in your config.
+More information about how to use the DEV-TOOLS can be found in [this](https://webdriver.io/blog/2019/09/16/devtools.html) blog post.
+
```

Comment:
Let's not encourage users anymore to use this.
```suggestion
```

Reply:
OK, I'll also remove it from the tests in the repo itself